src/model/fp.ml

```diff
@@ -0,0 +1,126 @@
+(* This file is free software, part of dolmen. See file "LICENSE" for more information *)
+
+(* Value definition *)
+(* ************************************************************************* *)
+open Farith
+
+let ops_rm =
+  Value.ops
+    ~compare:Mode.compare
+    ~print:Mode.pp
+    ()
+
+let ops =
+  Value.ops
+    ~compare:(fun b b' -> F.compare b b')
+    ~print:(fun fmt b -> Format.fprintf fmt "%a" F.pp b)
+    ()
```

Comment:
Event if they're just aliases, can you add type definitions for the rounding mode and fp numbers, with a small comment stating that we use the exact same representation as  the `Farith` lib ?

Reply:
even a `type t = F.t` would be enough, it's more for readability than anything

Reply:
> Ok. ( on another topic, do you think you could make a release of the current master? )

Sorry about the delay, I've done the release (all of the github side is done, still waiting on the merge into opam-repo, see ocaml/opam-repository#21490 )